plasmapy/formulary/collisions/frequencies.py

```diff
@@ -25,125 +24,144 @@
 
 
 class SingleParticleCollisionFrequencies:
-    @particles.particle_input
-    @validate_quantities(
-        v_drift={"can_be_negative": False},
-        T_b={
-            "can_be_negative": False,
-            "equivalencies": u.temperature_energy(),
-        },
-        n_b={"can_be_negative": False},
-    )
-    def __init__(
-        self,
-        test_particle: particles.ParticleLike,
-        field_particle: particles.ParticleLike,
-        *,
-        v_drift: u.m / u.s,
-        T_b: u.K,
-        n_b: u.m**-3,
-        Coulomb_log: u.dimensionless_unscaled,
-    ):
-        r"""
-        Compute collision frequencies between test particles
-        (labeled 'a') and field particles (labeled 'b').
+    r"""
+    Compute collision frequencies between test particles (labeled 'a')
+    and field particles (labeled 'b').
```

Comment:
Putting the docstring right after the class definition rather than in `__init__` to match what we've typically done elsewhere.  It probably doesn't change how it shows up in the docs.